core/src/test/scala/org/apache/spark/scheduler/SparkListenerSuite.scala

```diff
@@ -456,7 +456,9 @@ class SparkListenerSuite extends SparkFunSuite with LocalSparkContext with Match
     val conf = new SparkConf().setMaster("local").setAppName("test")
       .set(EXTRA_LISTENERS, listeners.map(_.getName))
     sc = new SparkContext(conf)
-    sc.listenerBus.listeners.asScala.count(_.isInstanceOf[BasicJobCounter]) should be (1)
+    // sc.listenerBus.listeners.asScala.count(_.isInstanceOf[BasicJobCounter]) should be (1)
+    assert(sc.listenerBus.listeners.asScala.count(_.isInstanceOf[BasicJobCounter]) == 1,
+      s"listeners = ${sc.listenerBus.listeners.asScala.mkString(", ")}")
```

Comment:
This change, and the change to LogUrlsStaqndaloneSuite was to debug why these assertions failed.
But looks like it is just flakey test probably (on my local box) - so removing them.